src/ripple/protocol/STAmount.h

```diff
@@ -104,6 +104,32 @@ class STAmount final
 
     STAmount (Issue const& issue, int mantissa, int exponent = 0);
 
+    std::size_t
+    size_of() const override
+    {
+        return sizeof(*this);
+    }
+
+    STBase*
+    copy (std::size_t n, void* buf) const override
```

Comment:
@HowardHinnant  I'm open to suggestions on how to eliminate this code duplication